en: split troubleshoot to several docs (#470)
pingcap/docs-tidb-operator: en/exceptions.md

        ```

4. If the TiKV store with the latest `lastHeartbeatTime` that corresponds to a Pod is in a `Tombstone` state, it means that the offline process is completed. At this time, you need to re-create the Pod and bind it with a new PV to perform recovery by taking the following steps:

    1. Set the `reclaimPolicy` value of the PV corresponding to the store to `Delete`:

        {{< copyable "shell-regular" >}}

        ```shell
        kubectl patch $(kubectl get pv -l app.kubernetes.io/instance=${release_name},tidb.pingcap.com/store-id=${store_id} -o name) -p '{"spec":{"persistentVolumeReclaimPolicy":"Delete"}}
        ```

    2. Remove the PVC used by the Pod:

        {{< copyable "shell-regular" >}}

        ```shell
        kubectl delete -n ${namespace} pvc tikv-${pod_name} --wait=false
        ```

    3. Remove the Pod, and wait for it to be re-created:

        {{< copyable "shell-regular" >}}

        ```shell
        kubectl delete -n ${namespace} pod ${pod_name}
        ```

    After the Pod is re-created, a new store is registered in the cluster. Then the recovery is completed.

> ## Long queries are abnormally interrupted in TiDB

Reviewer: ```suggestion
## Persistent connections are abnormally terminated in TiDB
```

Load balancers often set the idle connection timeout. If no data is sent via a connection for a specific period of time, load balancer closes the connection.

> If a long query is interrupted when you use TiDB, check the middleware program between the client and the TiDB server.

Reviewer: ```suggestion
If a persistent connection is terminated when you use TiDB, check the middleware program between the client and the TiDB server.
```

> If a persistent connection is terminated when you use TiDB, check the middleware program between the client and the TiDB server.
> If the idle timeout is not long enough for your query, try to set the timeout to a larger value. If you cannot reset it, enable the `tcp-keep-alive` option in TiDB.

Reviewer 2: ```suggestion
- If a persistent connection is terminated when you use TiDB, check the middleware program between the client and the TiDB server.
- If the idle timeout is not long enough for your query, try to set the timeout to a larger value. If you cannot reset it, enable the `tcp-keep-alive` option in TiDB.
```
